.github/workflows/macos.yaml

```diff
@@ -3,36 +3,42 @@ name: macos
 on: [push, pull_request]
 
 jobs:
-
   macos-build:
-    runs-on: ${{ matrix.macos-version }}
+    runs-on: ${{ matrix.os.os }}
     strategy:
       fail-fast: false
       matrix:
-        macos-version:
-          - 'macos-latest'
+        os:
+          - {arch: clang_64, os: macos-13}
+          - {arch: arm64, os: macos-latest}
 
     steps:
       - name: Sync repository
         uses: actions/checkout@v2
 
-      - name: Cache Qt
-        id: cache-qt
-        uses: actions/cache@v1  # not v2!
-        with:
-          path: '${{ github.workspace }}/qt_installation/'
-          key: ${{ runner.os }}-QtCache
+      - name: Clean Homebrew locks and cache
+        run: |
+          rm -rf /usr/local/var/homebrew/locks
+          brew cleanup -s
```

Comment:
you can likely append this to the next step so you have everything homebrew related in a single step